packages/x-components/src/utils/focus.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Comma separated list of common focusable selectors.
+ */
```

Comment:
The build is now failing due to this file being exported and this doc missing a `release tag`. Can you add the `@internal` tag to it? I think it should be `@internal`, but if you think otherwise feel free to say it.

```suggestion
/**
 * Comma separated list of common focusable selectors.
 *
 * @internal
 */
```